### DISCUSSION TITLE
Problem with corepack cache and yarn 3+

### DISCUSSION THREAD

**Discussion by JP250552** [renovatebot/renovate#27462] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

There appear to be some missing files related to corepack. There are no failures in the dashboard logs and nothing from what I could tell that appears to be related this this error. 

See this minimal reproduction:
https://github.com/JP250552/renovate-test/pull/2

```
Internal Error: ENOENT: no such file or directory, open '/home/ubuntu/.cache/node/corepack/lastKnownGood.json'
Error: ENOENT: no such file or directory, open '/home/ubuntu/.cache/node/corepack/lastKnownGood.json'
```

![image](https://github.com/renovatebot/renovate/assets/75643994/212d55b0-84f7-44b6-883d-93a16420c57b)

### Relevant debug logs

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment, marked as answer]
Latest corepack has a bug: https://github.com/nodejs/corepack/issues/379

See https://github.com/renovatebot/renovate/discussions/27452 for workaround

**Reply by JP250552** [reply]
Thank you for the quick response!
